src/styles/product-card.scss

```diff
@@ -0,0 +1,136 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
```

Comment:
move to `*` selector and delete in all other places

Reply:
> move to `*` selector and delete in all other places

I can't understand your message. That piece of code I am use once.

Reply:
> > move to `*` selector and delete in all other places
> 
> I can't understand your message. That piece of code I am use once.
box-sizing always use in * selector